deploy/helm/flightctl-ui/templates/flightctl-ui-service.yaml

```
@@ -2,11 +2,13 @@ kind: Service
 apiVersion: v1
 metadata:
   name: flightctl-ui
-  namespace: {{ .Values.flightctlUi.namespace }}
+  namespace: {{ .Release.Namespace }}
   labels:
     app: flightctl-ui
+  {{- if not ((.Values.global).baseDomainTls).cert }}
```

```suggestion
  {{- if not ((.Values.global).baseDomainTls).cert }}
```

Why is this inverted everywhere, shouldn't it be:

  {{- if ((.Values.global).baseDomainTls).cert }}

?

May be I just need more coffee ':D

---

also confused with the instances in this file : https://github.com/flightctl/flightctl-ui/pull/112/files#diff-325541283d9f28d1334be3088b93999822f1624d9b7a435c54c76c6aa4a7a564R79

---

This condition means:
if user DOES NOT specify their own TLS cert (via `global.baseDomainTls`), we will annotate the Service and let OpenShift create a cert for us.

---

You're right about the second one, the condition should not be there at all.